sys-devel/gcc/README.md

```diff
@@ -0,0 +1,11 @@
+# Flatcar Container Linux notes for the GCC package
+
+This readme captures notes for upgrading GCC in the SDK toolchain.
+
+Starting notes
+- The SDK uses cross compiler environments for all borard, even "native" (x86 on x86).
```

Comment:
s/borard/board/

Reply:
OH NO! I missed the opportunity to write "Borat" :smile: Will fix.